src/components/members/detail/ActivitySection/MemberDetail.tsx

```diff
@@ -144,13 +91,7 @@ const MemberDetail: FC<MemberDetailProps> = ({ memberId }) => {
     }
   }, [profile, memberId]);
 
-  useEffect(() => {
-    if (isVisible) {
-      fetchNextPage();
-    }
-  }, [isVisible, fetchNextPage]);
-
-  if (error?.response?.status === 400) {
+  if (profileError?.response?.status === 400 || crewError?.response?.status === 400) {
```

Comment:
```suggestion
if (profileError?.response?.status === 400 || (axios.isAxiosError(crewError) && crewError.response?.status === 400)) {
    return <EmptyProfile />;
  }
```

다른 코드들을 최대한 변경하지 않고 `AxiosError` 인지만 검사해야 한다면 요렇게 하면 간단히 해결할 수 있을 것 같아요! (요건 타입가드를 이용한 방법입니당)

말씀주신대로 제네릭으로 주입시켜줘도 괜찮을 것 같구요!

Reply:
오와... 저도 알아갑니다... 최고에요!!! 🚀🚀🚀

Reply:
아 타입가드를 활용하는 방법도있겠군요..!!!!! 감사합니다 요게 제일 빠른 해결책이 될 것 같아요